datamover/pkg/drivers/https/migration.go

```diff
@@ -555,6 +574,22 @@ func getOsdsS3Objs(ctx context.Context, conn *pb.Connector, filt *pb.Filter,
 	return srcObjs, nil
 }
 
+func getIBMCosObjs(ctx context.Context, conn *pb.Connector, filt *pb.Filter,
+	defaultSrcLoca *LocationInfo) ([]*osdss3.Object, error) {
+	//TODO:need to support filter
```

Comment:
Can you add your github ID next to TODO?  i.e., // TODO(acorbellini): need to ...

Reply:
Changed the TODO to a different text. This code needs to reuse getAWSS3Objs, but that function call the s3Mover functionality.